cypress/support/accessibility/a11y-utils.ts

```diff
@@ -125,6 +125,9 @@ export default (from: number, end: number) => {
       !prepareUrl[0].startsWith("breadcrumbs") &&
       !prepareUrl[0].startsWith("tabs") &&
       !prepareUrl[0].startsWith("tooltip") &&
+      !prepareUrl[0].startsWith("simple-select") &&
```

Comment:
we don't need to specify each `select` component because the url is `select-filterable`, `select-multiple` and `select`. So 1 `select` will be enough.

```suggestion
      !prepareUrl[0].startsWith("select") &&
```